navigation/index.tsx

```diff
@@ -179,50 +263,43 @@ function BottomTabNavigator() {
         tabBarInactiveTintColor: Colors[colorScheme].tabIconDefault,
         tabBarStyle: {
           backgroundColor: Colors[colorScheme].menu,
-          height: "11%",
+          height: "10%",
           padding: 15,
         },
       }}
     >
       <BottomTab.Screen
         name="Home"
-        component={StartScreen}
-        options={({ navigation }: RootTabScreenProps<"Home">) => ({
+        component={HomeStackNavigator}
+        options={{
           title: "",
-          headerTitle: "Hem",
+          headerShown: false,
           tabBarIcon: ({ color }) => (
             <Ionicons name="ios-home-outline" size={24} color={color} />
           ),
-        })}
+        }}
       />
+
       <BottomTab.Screen
         name="Search"
-        component={SearchScreen}
+        component={SearchStackScreen}
         options={{
           title: "",
-          // headerShown: false,
+          headerShown: false,
           headerStyle: { height: Constants.statusBarHeight },
           tabBarIcon: ({ color }) => (
             <AntDesign name="search1" size={24} color={color} />
           ),
         }}
       />
-      <BottomTab.Screen
-        name="Notifications"
-        component={NotificationsScreen}
-        options={{
-          title: "",
-          tabBarIcon: ({ color }) => (
-            <Ionicons name="notifications-outline" size={24} color={color} />
-          ),
-        }}
-      />
+
       <BottomTab.Screen
         name="Profile"
         component={ProfileScreen}
```

Comment:
I think we should add ProfileStackScreen as well. When I go to profile and then click on a product I am navigated to HomeStack which leads to the back button takes me to HomeScreen and not back to ProfileScreen.

Reply:
And all names on BottomTab.Screens should be changed to avoid error message: *Fined two screens with the same name nested inside each other...*